TCAT/Controllers/RouteDetail+ContentViewController.swift

```diff
@@ -32,9 +29,11 @@ class RouteDetailContentViewController: UIViewController, GMSMapViewDelegate, CL
     private var liveTrackingNetworkRefreshRate: Double = LiveIndicator.INTERVAL * 1.0
     private var liveTrackingNetworkTimer: Timer?
     private var locationManager = CLLocationManager()
+    private var mapView: GMSMapView!
     private let networking: Networking = URLSession.shared.request
     private var paths: [Path] = []
     private var route: Route!
+    private var routeOptionsCell: RouteTableViewCell?
```

Comment:
why do we need to store a reference to this?

Reply:
It was being used to take a screenshot of the cell when a user shares but I'll make that a delegate function n just handle that in the main vc

Reply:
Wait actually I'm not sure if we can do that. Cuz it takes an image of the route options cell which is like an overview of the route and then the user can share that specific route with someone else within the route details page. I'll think of a better way to do this but I'll probably leave this for now if that's ok